app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -37,8 +37,7 @@ params:
     - name: handle_unknown
       default: "`false`"
       required: false
-      description: Allow transform to apply to unmatched Service, Route, or Workspace
-      (404) responses. This flag only applies to one configuration entry.
```

Comment:
Have requested clarification from Lluis on whether the flag applies to both unknown and expected because there is a discrepancy per the readme. Also, the template has issues with line 41 column 7 no matter how I format the additional information. Will add to a note section under params table after Lluis confirms.